src/Pyz/Client/Catalog/CatalogDependencyProvider.php

```diff
@@ -208,4 +224,68 @@ protected function getProductConcreteCatalogSearchQueryExpanderPlugins(): array
             new MerchantProductReferenceQueryExpanderPlugin(),
         ];
     }
+
+    /**
+     * @return array<string, array<\Spryker\Client\Catalog\Dependency\Plugin\FacetConfigTransferBuilderPluginInterface>>
+     */
+    protected function getFacetConfigTransferBuilderPluginVariants(): array
+    {
+        return [
+            SearchHttpConfig::TYPE_SEARCH_HTTP => [
+                new CategoryFacetConfigTransferBuilderPlugin(),
+                new PriceFacetConfigTransferBuilderPlugin(),
+                new RatingFacetConfigTransferBuilderPlugin(),
+                new ProductLabelSearchHttpFacetConfigTransferBuilderPlugin(),
+            ],
+        ];
+    }
+
+    /**
+     * @phpstan-return array<\Spryker\Client\SearchExtension\Dependency\Plugin\QueryInterface>
+     *
+     * @return array<\Spryker\Client\Search\Dependency\Plugin\QueryInterface>
+     */
+    protected function createCatalogSearchQueryPluginVariants(): array
+    {
+        $searchContextTransfer = (new SearchContextTransfer())
+            ->setSourceIdentifier(SearchHttpConfig::SOURCE_IDENTIFIER_PRODUCT)
+            ->setSearchHttpContext(new SearchHttpSearchContextTransfer());
+
+        return [
+            new SearchHttpQueryPlugin($searchContextTransfer),
+        ];
+    }
+
+    /**
+     * @return array<string, array<\Spryker\Client\SearchExtension\Dependency\Plugin\QueryExpanderPluginInterface>>
+     */
+    protected function createCatalogSearchQueryExpanderPluginVariants(): array
+    {
+        return [
+            SearchHttpConfig::TYPE_SEARCH_HTTP => [
+                new BasicSearchHttpQueryExpanderPlugin(),
+                new ProductPriceSearchHttpQueryExpanderPlugin(),
+                new FacetSearchHttpQueryExpanderPlugin(),
```

Comment:
+  `new MerchantReferenceSearchHttpQueryExpanderPlugin(),`

Reply:
I'm sorry but now it's not needed anymore :)

https://github.com/spryker/suite-nonsplit/pull/7744/files
but requires SearchHttp upgrade 0.3.1 >> 0.3.2


cc @Maksym-Ohnianikov

Reply:
Updated
Do we want another QA round?

Reply:
as you wish